examples/shopping-cart/components/ShoppingCart.vue

```diff
@@ -3,7 +3,9 @@
     <h2>Your Cart</h2>
     <p v-show="!products.length"><i>Please add some products to cart.</i></p>
     <ul>
-      <li v-for="product in products">
+      <li
+        v-for="product in products"
+        :key="product.id">
```

Comment:
Hi, I'm new to Vue, here the product seems to have no id property, i'm not sure if i'm correct or wrong.